data/datasets.json

```diff
@@ -1,15 +1,15 @@
 {
     "neuroglancer": {
-        "Connectomics_Department": {
-            "drosophila": {
+        "Demo_Lab": {
```

Comment:
@normanrz: Currently `Connectomics_Department` is the default Organization in webknossos, so let's stick to that, as long as we don't change it in webknossos.

Reply:
see #58